content/karpenter/050_karpenter/set_up_the_environment.md

```diff
@@ -14,7 +14,7 @@ Instances launched by Karpenter must run with an InstanceProfile that grants per
 Karpenter also utilizes an SQS queue and EventBridge rules to handle Spot Interruption Notifications and AWS Health Events.
 
 ```
-export KARPENTER_VERSION=v0.23.0
+export KARPENTER_VERSION=v0.26.1
```

Comment:
@albernazj93 thanks you for your contribution :) ... I have some questions for you.

Can we update it to the latest version? `v0.27.3` ... there's also a link to the CloudFormation template in this same file pointing to `v0.22.1` ... also, is there anything relevant with this version updates that we might need to update within the workshop instructions?

Reply:
Hi @chrismld no problem at all :)
Excellent catch there on the URL pointing to v0.22.1. I have just pushed a fix to that.
Regarding the latest version, I don't think so since I ran the workshop using the 0.26.1. I could not find any breaking changes while I was testing it. 

Would please review my latest push? The URL should be fixed now. Cheers!